homework-classes/Repository.js

```diff
@@ -13,8 +12,9 @@ class Repository {
    * @param {HTMLElement} container The container element in which to render the repository.
    */
   render(container) {
-    // TODO: replace the next line with your code.
-    Util.createAndAppend('pre', container, { text: JSON.stringify(this.repository, null, 2) });
+    Util.createAndAppend('option', container, {
```

Comment:
You'll have to add more here. Think you can copy code over from the previous week? Also, can't read if not white colored.